api/platformpb/json/client/platform/platform_client.go

```diff
@@ -63,6 +65,39 @@ func (a *Client) Connect(params *ConnectParams) (*ConnectOK, error) {
 	return nil, runtime.NewAPIError("unexpected success response: content available as default response in error", unexpectedSuccess, unexpectedSuccess.Code())
 }
 
+/*
+  Disconnect disconnects disconnects a PMM server from the organization created on percona portal
```

Comment:
please check description

Reply:
This was generated. For some reason it causing this. Changed sentence to avoid this problem.